js/videoBlock.js

```diff
@@ -0,0 +1,51 @@
+const video = [
+    {
+        title: "The youngest Beatles tribute band",
+        frameborder: 0,
+        accelerometer: true,
+        encryptedMedia: true,
+        gyroscope: true,
+        pictureInPicture: true,
+        fullscreen: true,
+    },
+    {
+        title: "Teen Millionaires - successful businesses run by teens",
+        frameborder: 0,
+        accelerometer: true,
+        encryptedMedia: true,
+        gyroscope: true,
+        pictureInPicture: true,
+        fullscreen: true,
+    },
+]
+
+let source = ["https://www.youtube.com/embed/Uq-FOOQ1TpE", "https://www.youtube.com/embed/93mctUlDofw", "https://www.youtube.com/embed/1YCGM7FiCGg", "https://www.youtube.com/embed/vPukHebsQeQ", "https://www.youtube.com/embed/h11u3vtcpaY"]
+
+function displayVideos (video) {
+    for (let i=0; i<video.length; i++ )
+    let videoEl = document.getElementById("video");
+    video.src = url;
```

Comment:
where you get this url value from?

Reply:
changed it to: `videos.src = source[i].value;`
but not right as not working...